pegen/metagrammar.gram

```diff
@@ -0,0 +1,55 @@
+start: rules ENDMARKER {Rules(rules)}
+
+rules:
+    | rule rules { [rule] + rules }
+    | rule { [rule] }
+
+rule:
+    | rulename ":" alts NEWLINE INDENT more_alts DEDENT { Rule(*rulename, alts + more_alts) }
```

Comment:
I betcha this line doesn't work -- it's trying to add two `Rhs` instances.

I recommend just dropping it for now.

Reply:
I can do the same as I did in `more_alts`:

```
Rhs(alts.alts + more_alts.alts)
```

Reply:
Sure.